game/pages_admin/admin_queue.php

```diff
@@ -149,11 +149,11 @@ function Admin_Queue ()
 
     <br/>
     <form action="index.php?page=admin&session=<?=$session;?>&mode=Queue" method="POST">
-    Показать задания игрока: <input size=15 name="player" value="<?=$playername;?>">
```

Comment:
Then ADM_QUEUE_LOOKUP should be added to the other locas as well. I can do it if you don't want to do it yourself.